cmu_cs3_graphics.py

```diff
@@ -41,27 +41,12 @@ def __setattr__(self, attr, value):
         else:
             return super().__setattr__(attr, value)
 
-def callUserFn(self, fnName, args):
-    # replacement for App.callUserFn()
-    fnName0 = fnName
-    if app.hasException: return
-    if app.mode not in [None, '']:
-        fnName = app.mode + fnName[0].upper() + fnName[1:]
-    if fnName in self.userGlobals:
-        (self.userGlobals[fnName])(app.appWrapper, *args)
-        if (not fnName0.endswith('redrawAll')): redrawAllWrapper(self)
-
-def redrawAllWrapper(app):
-    app.group.clear()
-    try: app.inRedrawAll = True; callUserFn(app, 'redrawAll', [ ])
-    finally: app.inRedrawAll = False
-
 def runApp(width=400, height=400, **kwargs):
     kwargs.update({'width': width, 'height': height})
     for kw in kwargs:
         setattr(app.appWrapper, kw, kwargs[kw])
-    callUserFn(app, 'onAppStart', [ ])
-    redrawAllWrapper(app)
+    app.callUserFn('onAppStart', [ ])
```

Comment:
Could you modify things so that this file is included in the pip & zip packages? I think it is not in pip right now at least. Not a big deal, but I think we are tracking that as a bug somewhere.

Reply:
I think this should probably be its own task. Including `cmu_cs3_graphics` in the pip package won't do much good on its own, because it just hides the file in your libs somewhere. Same with the zip package -- we need new documentation on how to use the new file, and ideally we'd restructure some things so you could run a line like `from cmu_graphics.cs3 import *` instead of pulling this file outside of the `cmu_graphics` folder.

Reply:
Fwiw, if you run the old `cmu_cs3_graphics` file with these new changes, everything still works. You just don't get the benefit of `_safeMethod`

Reply:
This all sounds reasonable.